src/main/resources/static/json/org.json

```diff
@@ -1,21 +0,0 @@
-{"ID":1,
-"Name":"ABC",
-"Head":"Mumbai",
-"Count":"100"
```

Comment:
@Rinshakp ,

What is the use of this JSON?
Is this the correct format of JSON?